docs/docs/zh/guide/upgrade/upgrade.md

```diff
@@ -59,7 +59,7 @@ jar 包 并添加到 `./tools/libs` 目录下，设置以下环境变量
 
 执行结果：
 
-- 原血缘数据迁移至新血缘表 `t_ds_process_task_lineage`。
+- 原血缘数据迁移至新血缘表 `t_ds_workflow_task_lineage`。
```

Comment:
Does upgrade.md versionable? or only have single version in our document just like faq.md?

Reply:
Yes. It's versionable.